From f5ab857c84389d4d56bce65daecf22b99a49ef80 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Storsj=C3=B6?= <martin@martin.st>
Date: Fri, 25 Aug 2017 12:46:32 +0300
Subject: [PATCH] Ignore --whole-archive, --no-whole-archive and --sysroot, fix
 the --stack option

This is necessary to pass some cmake compiler checks.
---
 MinGW/Options.td | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/MinGW/Options.td b/MinGW/Options.td
index abf53c9..569ee1b 100644
--- a/MinGW/Options.td
+++ b/MinGW/Options.td
@@ -15,7 +15,7 @@ def l: JoinedOrSeparate<["-"], "l">, MetaVarName<"<libName>">,
   HelpText<"Root name of library to use">;
 def shared: F<"shared">, HelpText<"Build a shared object">;
 def subs: Separate<["--"], "subsystem">, HelpText<"Specify subsystem">;
-def stack: Flag<["--"], "stack">, Alias<stack>;
+def stack: Separate<["--"], "stack">;
 def outlib: Separate<["--"], "out-implib">, HelpText<"Import library name">;
 
 // Currently stubs to avoid errors
@@ -29,6 +29,9 @@ def O: Joined<["-"], "O">, HelpText<"Optimize output file size">;
 def v: Flag<["-"], "v">, HelpText<"Display the version number">;
 def verbose: F<"verbose">, HelpText<"Verbose mode">;
 def version: F<"version">, HelpText<"Display the version number and exit">;
+def whole_archive: F<"whole-archive">;
+def no_whole_archive: F<"no-whole-archive">;
+def sysroot: J<"sysroot">, HelpText<"Sysroot">;
 
 // Alias
 def alias_entry_e: JoinedOrSeparate<["-"], "e">, Alias<entry>;
-- 
2.7.4

